app/views/tenants/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<% @tenants.each do |tenant| %>
+  <p><%= tenant.organization %></p>
+<% end %>
```

Comment:
Maybe we don't need this? Adam already added the tenant index view, nested under the admin (admin/tenants/index.html.erb), which makes sense to me.